tailwind.config.js

```diff
@@ -1,8 +1,9 @@
 module.exports = {
   future: {
+    purgeLayersByDefault: true,
     removeDeprecatedGapUtilities: true
   },
-  purge: [],
+  purge: ['./src/**/*.{js,ts,jsx,tsx}'],
```

Comment:
We have to keep in eye on this because it used to give us issues before https://github.com/debtcollective/home/pull/8 nevertheless, this is other approach

Reply:
The preview works but I'll roll back the deploy if it fails in prod,